packages/contrast-colors/index.js

```diff
@@ -389,7 +679,14 @@ function filterNaN(x) {
 }
 
 // Helper function for rounding color values to whole numbers
-function fixColorValue(color, format, object = false) {
+function convertColorValue(color, format, object = false) {
+  if(!color) {
+    throw new Error(`Cannot convert color value of ${color}`)
+  }
+  if(!format) {
+    throw new Error(`Cannot convert to colorspace ${format}`)
+  }
+
   let colorObj = colorSpaces[format].function(color);
```

Comment:
You probably need to check if `colorSpaces[format] !== undefined` here and throw accordingly.